CHANGELOG.md

```diff
@@ -3,7 +3,9 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## Version [1.1.1][unreleased] - (in development)
-- TBD
+- Changed: Dependency updates
+- Changed: Add application header
```

Comment:
I don't see where this is being added, maybe missed adding a file to the commit?